tests/pom.xml

```diff
@@ -44,6 +44,16 @@
   </modules>
 
   <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${version.maven-deploy-plugin}</version>
```

Comment:
Same as above: remove the `<version>` and add `<groupId>org.apache.maven.plugins</groupId>`